src/config/config.service.ts

```diff
@@ -81,31 +81,31 @@ export class ConfigService {
   }
 
   public getBlockchainScanIntervalMinutes(): number {
-    return this.nestConfigService.get<number>('BLOCKCHAIN_SCAN_INTERVAL_MINUTES') ?? 1;
```

Comment:
This change adds explicit parseInt() calls to ensure numeric environment 
variables are properly converted to numbers, maintaining type safety and 
fixing failing unit tests.